diff --git a/2021-11-03/index.d.ts b/2021-11-03/index.d.ts
index 738c8682307129927087fdd023954c576a3b9081..1a0a5734ecf954bbc8eb3a5fe5a769d56f053d80 100644
--- a/2021-11-03/index.d.ts
+++ b/2021-11-03/index.d.ts
@@ -2244,7 +2244,7 @@ declare interface IncomingRequestCfPropertiesExportedAuthenticatorMetadata {
  * Geographic data about the request's origin.
  */
 declare type IncomingRequestCfPropertiesGeographicInformation =
-  | {}
+  | Record<string, never>
   | {
       /** The country code `"T1"` is used for requests originating on TOR  */
       country: "T1";
diff --git a/2021-11-03/index.ts b/2021-11-03/index.ts
index 01cac3aa2bebcdd7c238e850c90781d7d9982561..535c9ae82ef4f5b04b33472dbdd54d1ce80f45ed 100644
--- a/2021-11-03/index.ts
+++ b/2021-11-03/index.ts
@@ -2249,7 +2249,7 @@ export interface IncomingRequestCfPropertiesExportedAuthenticatorMetadata {
  * Geographic data about the request's origin.
  */
 export type IncomingRequestCfPropertiesGeographicInformation =
-  | {}
+  | Record<string, never>
   | {
       /** The country code `"T1"` is used for requests originating on TOR  */
       country: "T1";
diff --git a/2022-01-31/index.d.ts b/2022-01-31/index.d.ts
index 33afac4929883f6c3f8183330003bcfa2863104d..cdc473c72708d30c33f2f366b7899926a9a5d05f 100644
--- a/2022-01-31/index.d.ts
+++ b/2022-01-31/index.d.ts
@@ -2230,7 +2230,7 @@ declare interface IncomingRequestCfPropertiesExportedAuthenticatorMetadata {
  * Geographic data about the request's origin.
  */
 declare type IncomingRequestCfPropertiesGeographicInformation =
-  | {}
+  | Record<string, never>
   | {
       /** The country code `"T1"` is used for requests originating on TOR  */
       country: "T1";
diff --git a/2022-01-31/index.ts b/2022-01-31/index.ts
index c2678b7b95d58458ef9c4e5324c691b6587b9737..38cd6dbdbb0d25784844041ca5e6c360e7d75cd6 100644
--- a/2022-01-31/index.ts
+++ b/2022-01-31/index.ts
@@ -2235,7 +2235,7 @@ export interface IncomingRequestCfPropertiesExportedAuthenticatorMetadata {
  * Geographic data about the request's origin.
  */
 export type IncomingRequestCfPropertiesGeographicInformation =
-  | {}
+  | Record<string, never>
   | {
       /** The country code `"T1"` is used for requests originating on TOR  */
       country: "T1";
diff --git a/2022-03-21/index.d.ts b/2022-03-21/index.d.ts
index 3dad899140682e4ccd516c98db2f84a5e0390b47..7e2e340756d9eee96e01e3cdf1f31922250b7a0e 100644
--- a/2022-03-21/index.d.ts
+++ b/2022-03-21/index.d.ts
@@ -2236,7 +2236,7 @@ declare interface IncomingRequestCfPropertiesExportedAuthenticatorMetadata {
  * Geographic data about the request's origin.
  */
 declare type IncomingRequestCfPropertiesGeographicInformation =
-  | {}
+  | Record<string, never>
   | {
       /** The country code `"T1"` is used for requests originating on TOR  */
       country: "T1";
diff --git a/2022-03-21/index.ts b/2022-03-21/index.ts
index 4c3ce0b93b589ebb5e60004ada005a760c73dacb..ee3e04370e975b29a8165adf7317d1edd16101ad 100644
--- a/2022-03-21/index.ts
+++ b/2022-03-21/index.ts
@@ -2241,7 +2241,7 @@ export interface IncomingRequestCfPropertiesExportedAuthenticatorMetadata {
  * Geographic data about the request's origin.
  */
 export type IncomingRequestCfPropertiesGeographicInformation =
-  | {}
+  | Record<string, never>
   | {
       /** The country code `"T1"` is used for requests originating on TOR  */
       country: "T1";
diff --git a/2022-08-04/index.d.ts b/2022-08-04/index.d.ts
index 24e89a2ff99538fc49ee7f5cd2f5291812f12214..f941d849f82393082b75de16724e294733d3ee68 100644
--- a/2022-08-04/index.d.ts
+++ b/2022-08-04/index.d.ts
@@ -2237,7 +2237,7 @@ declare interface IncomingRequestCfPropertiesExportedAuthenticatorMetadata {
  * Geographic data about the request's origin.
  */
 declare type IncomingRequestCfPropertiesGeographicInformation =
-  | {}
+  | Record<string, never>
   | {
       /** The country code `"T1"` is used for requests originating on TOR  */
       country: "T1";
diff --git a/2022-08-04/index.ts b/2022-08-04/index.ts
index 439f5e8562f92bb8d96792a28d2342c7e33e8a6a..e9f1cc115ae98ea4d3ce06acf831fcb03880cd9f 100644
--- a/2022-08-04/index.ts
+++ b/2022-08-04/index.ts
@@ -2242,7 +2242,7 @@ export interface IncomingRequestCfPropertiesExportedAuthenticatorMetadata {
  * Geographic data about the request's origin.
  */
 export type IncomingRequestCfPropertiesGeographicInformation =
-  | {}
+  | Record<string, never>
   | {
       /** The country code `"T1"` is used for requests originating on TOR  */
       country: "T1";
diff --git a/2022-10-31/index.d.ts b/2022-10-31/index.d.ts
index 6582d60916882970de8017011c36b1f0fd093b7b..d5d0b6965864577a257af3bf15ec33bad59df920 100644
--- a/2022-10-31/index.d.ts
+++ b/2022-10-31/index.d.ts
@@ -2233,7 +2233,7 @@ declare interface IncomingRequestCfPropertiesExportedAuthenticatorMetadata {
  * Geographic data about the request's origin.
  */
 declare type IncomingRequestCfPropertiesGeographicInformation =
-  | {}
+  | Record<string, never>
   | {
       /** The country code `"T1"` is used for requests originating on TOR  */
       country: "T1";
diff --git a/2022-10-31/index.ts b/2022-10-31/index.ts
index ff659cf4e175ea11738d8fd7fd190c6d93c8a6be..472da2de0dce45a40b27d044e159ef342e79aef6 100644
--- a/2022-10-31/index.ts
+++ b/2022-10-31/index.ts
@@ -2238,7 +2238,7 @@ export interface IncomingRequestCfPropertiesExportedAuthenticatorMetadata {
  * Geographic data about the request's origin.
  */
 export type IncomingRequestCfPropertiesGeographicInformation =
-  | {}
+  | Record<string, never>
   | {
       /** The country code `"T1"` is used for requests originating on TOR  */
       country: "T1";
diff --git a/2022-11-30/index.d.ts b/2022-11-30/index.d.ts
index 2405b09617ae1c7683804e9260d7295d30c3ed2c..8c77037b83bc4cf3e35da8469c0d103c99e03102 100644
--- a/2022-11-30/index.d.ts
+++ b/2022-11-30/index.d.ts
@@ -2236,7 +2236,7 @@ declare interface IncomingRequestCfPropertiesExportedAuthenticatorMetadata {
  * Geographic data about the request's origin.
  */
 declare type IncomingRequestCfPropertiesGeographicInformation =
-  | {}
+  | Record<string, never>
   | {
       /** The country code `"T1"` is used for requests originating on TOR  */
       country: "T1";
diff --git a/2022-11-30/index.ts b/2022-11-30/index.ts
index 41b6ff86a2f4fc7ae60fa889128b54a1f0d3ee60..7fa54a8621691954176231809744fdc9db54b80e 100644
--- a/2022-11-30/index.ts
+++ b/2022-11-30/index.ts
@@ -2241,7 +2241,7 @@ export interface IncomingRequestCfPropertiesExportedAuthenticatorMetadata {
  * Geographic data about the request's origin.
  */
 export type IncomingRequestCfPropertiesGeographicInformation =
-  | {}
+  | Record<string, never>
   | {
       /** The country code `"T1"` is used for requests originating on TOR  */
       country: "T1";
diff --git a/experimental/index.d.ts b/experimental/index.d.ts
index cdc09b6280b573380d972886f18bcbb5822aaf1e..b2bd020e4743ae430a191c7a712fc77675db22db 100644
--- a/experimental/index.d.ts
+++ b/experimental/index.d.ts
@@ -2237,7 +2237,7 @@ declare interface IncomingRequestCfPropertiesExportedAuthenticatorMetadata {
  * Geographic data about the request's origin.
  */
 declare type IncomingRequestCfPropertiesGeographicInformation =
-  | {}
+  | Record<string, never>
   | {
       /** The country code `"T1"` is used for requests originating on TOR  */
       country: "T1";
diff --git a/experimental/index.ts b/experimental/index.ts
index 968c5692624326285f2785a89af2b8cb0acf08ec..68175f0735f01c5c5d03f1a7e619494d5d408f23 100644
--- a/experimental/index.ts
+++ b/experimental/index.ts
@@ -2242,7 +2242,7 @@ export interface IncomingRequestCfPropertiesExportedAuthenticatorMetadata {
  * Geographic data about the request's origin.
  */
 export type IncomingRequestCfPropertiesGeographicInformation =
-  | {}
+  | Record<string, never>
   | {
       /** The country code `"T1"` is used for requests originating on TOR  */
       country: "T1";
diff --git a/index.d.ts b/index.d.ts
index 5c7299b1cace2d451690c6e6e376ef2024cdb9a0..cdee08fb5be5c28c028da0aff675f3aacf228510 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -2244,7 +2244,7 @@ declare interface IncomingRequestCfPropertiesExportedAuthenticatorMetadata {
  * Geographic data about the request's origin.
  */
 declare type IncomingRequestCfPropertiesGeographicInformation =
-  | {}
+  | Record<string, never>
   | {
       /** The country code `"T1"` is used for requests originating on TOR  */
       country: "T1";
diff --git a/index.ts b/index.ts
index eac5cb0f9ad5e48ba85bcf8095fbb439c0682192..db5154a398e97cd7951db3c305bba0532b7da69f 100644
--- a/index.ts
+++ b/index.ts
@@ -2249,7 +2249,7 @@ export interface IncomingRequestCfPropertiesExportedAuthenticatorMetadata {
  * Geographic data about the request's origin.
  */
 export type IncomingRequestCfPropertiesGeographicInformation =
-  | {}
+  | Record<string, never>
   | {
       /** The country code `"T1"` is used for requests originating on TOR  */
       country: "T1";
diff --git a/oldest/index.d.ts b/oldest/index.d.ts
index 5c7299b1cace2d451690c6e6e376ef2024cdb9a0..cdee08fb5be5c28c028da0aff675f3aacf228510 100644
--- a/oldest/index.d.ts
+++ b/oldest/index.d.ts
@@ -2244,7 +2244,7 @@ declare interface IncomingRequestCfPropertiesExportedAuthenticatorMetadata {
  * Geographic data about the request's origin.
  */
 declare type IncomingRequestCfPropertiesGeographicInformation =
-  | {}
+  | Record<string, never>
   | {
       /** The country code `"T1"` is used for requests originating on TOR  */
       country: "T1";
diff --git a/oldest/index.ts b/oldest/index.ts
index eac5cb0f9ad5e48ba85bcf8095fbb439c0682192..db5154a398e97cd7951db3c305bba0532b7da69f 100644
--- a/oldest/index.ts
+++ b/oldest/index.ts
@@ -2249,7 +2249,7 @@ export interface IncomingRequestCfPropertiesExportedAuthenticatorMetadata {
  * Geographic data about the request's origin.
  */
 export type IncomingRequestCfPropertiesGeographicInformation =
-  | {}
+  | Record<string, never>
   | {
       /** The country code `"T1"` is used for requests originating on TOR  */
       country: "T1";