diff --git a/dist/src/index.js b/dist/src/index.js
index cee85bc1677d6d7fd1f0027cda47d16a07b43116..1118bb81d4b66021d7fe76cb446f7a875aacd03c 100644
--- a/dist/src/index.js
+++ b/dist/src/index.js
@@ -139,7 +139,7 @@ var Statement = class {
     return this.#raw(statementWithBindings);
   }
   #raw(statementWithBindings) {
-    return statementWithBindings.raw();
+    return statementWithBindings.raw().all();
   }
 };
 