diff --git a/index.d.ts b/index.d.ts
index 907f8848a2ed71d8efb65f51f4111977ef6dbb1d..d9991a37dcb0f24f235fd2c1ab1831a9d09b9783 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -1846,6 +1846,11 @@ declare namespace React {
         | (string & {});
 
     interface HTMLAttributes<T> extends AriaAttributes, DOMAttributes<T> {
+        /**
+         * ONLY FOR USE WITH SATORI!!
+         */
+        tw?: string | undefined;
+
         // React-specific Attributes
         defaultChecked?: boolean | undefined;
         defaultValue?: string | number | ReadonlyArray<string> | undefined;